test/integration/dialects/postgres/sync-alter-enum.test.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const chai = require('chai'),
+  assert = chai.assert,
+  Support = require('../../support'),
+  Sequelize = Support.Sequelize,
+  dialect = Support.getTestDialect();
+
+if (dialect.match(/^postgres/)) {
+  describe('[POSTGRES Specific] sync with alter method with dataType enum', () => {
+    it('properly sync, #7649', async function() {
```

Comment:
To reduce nesting can you add something like the following here?

```
if (dialect !== 'postgres') {
   return;
```
(`getTestDialect` normalizes `postgres-native` to `postgres` iirc)

Reply:
Done.